### PR TITLE
Filter abilities

### PR DESCRIPTION
**Pull request by orichalque**

### CODE REVIEW

**Comment by orichalque** [comment]
- Normalized database
- Added filter button

File: index.html [hunk 1]
```diff
@@ -514,7 +514,7 @@ <h3>Final Modifier Deck</h3>
               <div class="row">
                 <div class="modifierContainer" v-for='card in modifiersChosen'>
                   <div class="col mt-3 " v-on:click="removeModifier(card)">
-                    <img v-bind:src="'data/'+card.image" style="width:8em" class="image rounded" />
+                    <img v-bind:src="'data/'+cprd.image" style="width:8em" class="image rounded" />
```
Comment (RaphaelPage0110) [inline index.html:517]: This looks like a typo

**Review by RaphaelPage0110 — CHANGES_REQUESTED** [review]
Looks good, there just seems to be typo in index.html line 517

**Comment by RaphaelPage0110** [comment]
image
Is there a way to not have the extra border here? (that's in the ability page)
The filter 'button' is not very pretty either. Maybe just a text would be better, to match the 'Character Level' label?

**Comment by RaphaelPage0110** [comment]
There is a bug on the Gear tab. Steps to reproduce : 
1) Go to the gear tab
2) Click on the 1-14 card back
Expected : the gears 1 to 14 shows
Actual : No gear is shown

I suspect it's because the filter is empty?

**Review by RaphaelPage0110 — APPROVED** [review]

**Pull request merged**